.eslintrc.js

```diff
@@ -67,7 +67,14 @@ module.exports = {
     // Enforces some naming conventions across the codebase.
     '@typescript-eslint/naming-convention': [
       2,
-      { selector: 'default', format: ['camelCase', 'UPPER_CASE', 'PascalCase'] },
+      {
+        selector: 'default',
+        format: ['camelCase', 'UPPER_CASE', 'PascalCase'],
+        filter: {
+          regex: '^(avatar_url|_json)$',
```

Comment:
I think using `eslint-disable-next-line @typescript-eslint/naming-convention` in places you have those variables would be a better solution. If we one day decide to remove one of those variables no one will remember that there is a related ESLint rule we need to modify as well.

Reply:
Okay, so I did as suggested.